Dockerfile

```diff
@@ -4,6 +4,9 @@ ENV ROS_DISTRO=kinetic
 # Dependencies for rosbridge
 RUN apt update && apt-get install -y xvfb firefox git wget ros-$ROS_DISTRO-rosbridge-server ros-$ROS_DISTRO-tf2-web-republisher ros-$ROS_DISTRO-common-tutorials ros-$ROS_DISTRO-rospy-tutorials ros-$ROS_DISTRO-actionlib-tutorials
 
+# Because of this issue https://github.com/ros/genpy/issues/132
+RUN apt update && apt-get upgrade -y
```

Comment:
Doing this tends to cost a lot of time and space, and makes images less consistent.  The base image should be updated soon.  I appreciate that this is needed for the tests to pass, I will merge it without this change.